utils/hare-bootstrap

```diff
@@ -242,6 +242,10 @@ EOF
     while read node _; do
         if is_localhost $node; then
             echo $node > $conf_dir/node-name
+            if [ $? -ne 0 ]
+                then
+                    echo "Permission Denied at" $node "for /var/lib/hare/node-name"
```

Comment:
Sorry, but it seems to me that this code will never be executed just because of `set -e` setting at the very beginning. If echo at previous line fails to write the data, the script will exit immediately with non-zero code.

Reply:
Um yeah. I just saw it. Yes this segment won't be executed.  Infact  if any line that fails, script will exit. Need to think of something else.

Reply:
Closing this PR. It was a Blunder. Sorry for the inconvenience caused.